x/go.mod

```diff
@@ -1,20 +1,21 @@
 module github.com/Jigsaw-Code/outline-sdk/x
 
-go 1.21
+go 1.22
```

Comment:
FYI, quic-go requires 1.22. We'll need to keep this quite recent in order to support newer protocol updates, unfortunately.